core/src/main/resources/hudson/model/AbstractItem/_api.jelly

```diff
@@ -30,8 +30,12 @@ THE SOFTWARE.
         To programmatically obtain the current configuration of this item (<code>config.xml</code>), access <a href="../config.xml" target="_blank">this URL</a> using <code>GET</code>.
         To programmatically update the configuration based on a modified <code>config.xml</code> file, you can send a <code>POST</code> request with <code>Content-Type: text/xml</code> header and the XML attached as request body to the <a href="../config.xml">same URL</a>.
     </p>
-    <h2>Delete a job</h2>
+    <h2>Rename an item</h2>
     <p>
-        To programmatically delete this job, send an HTTP <code>DELETE</code> request to <a href="../" target="_blank">this URL</a>.
+        To programmatically rename this item, send an HTTP <code>POST</code> request to <a href="../confirmRename" target="_blank">this URL</a> with a new name provided as a <code>newName</code> query parameter. A successful rename will respond with a 302 code with new URL of an item in the <code>location</code> header.
```

Comment:
@dwnusbaum Was this endpoint intended for use as API?

Reply:
🤷‍♂️ Not particularly, as far as I recall. I don't see anything wrong with using it as one though besides the fact that there are no guarantees about backwards compatibility, same as most random web random methods.

Reply:
> there are no guarantees about backwards compatibility, same as most random web random methods

If it gets documented here, it's kinda expected to remain compatible, which is why I'm asking.

Reply:
Yeah, I mean it doesn't look like anything major has changed since #3289 5 years ago, so take that as you will.